tests/utils.js

```diff
@@ -0,0 +1,9 @@
+const generateUUID = () => {
```

Comment:
why are you generating UUIDs? This is very uncommon. There is a widely accepted module called `uuid` that everyone uses.

If you were really intent on making your own implementation you would want to use `crypto.randomBytes()`

Reply:
Did know and googled how to generatte UUIDS.  This is only used for testing so I will install the UUID code and get rid of it.

Reply:
Done